library/common/network/configurator.cc

```diff
@@ -53,6 +53,17 @@
 #include <ifaddrs.h>
 #endif
 
+#if defined(__APPLE__)
+constexpr absl::string_view WlanPrefix = "en";
+constexpr absl::string_view WwanPrefix = "pdp_ip";
+#elif defined(__ANDROID_API__)
+constexpr absl::string_view WlanPrefix = "wlan";
+constexpr absl::string_view WwanPrefix = "rmnet";
+#else
+constexpr absl::string_view WlanPrefix = "";
```

Comment:
I would include a comment here saying that this is effectively disable filtering, not immediately clear from reading the code.